cmd/deploy/remote_test.go

```diff
@@ -350,7 +350,7 @@ func TestCreateDockerignore(t *testing.T) {
 			rdc := remoteDeployCommand{
 				fs: fs,
 			}
-			err := rdc.createDockerignore(tt.config.wd, tempDir)
+			err := rdc.createDockerignore(tt.config.wd, tempDir, "")
```

Comment:
```suggestion
			err := rdc.createDockerignore(tt.config.wd, tempDir, tt.config.manifestPathFlag)
```

WDYT if we add a unit test that at least exec the new logic?

```
{
			name: "with okteto manifest",
			config: config{
				wd:               dockerignoreWd,
				manifestPathFlag: "okteto.yml",
			},
		},

...

type config struct {
		wd               string
		manifestPathFlag string
	}
```

Reply:
updated and added test